cypress/page/menu-content-page.ts

```diff
@@ -1,25 +1,39 @@
 class MenuContentPage {
-  private tShirtMenu: string;
-  private menuContentPageURL: string;
-  private dressMenu: string;
+  menuContentPageURL: string;
 
   constructor() {
-    this.tShirtMenu = "li:nth-child(3) [title='T-shirts']";
-    this.menuContentPageURL = "http://automationpractice.com/";
-    this.dressMenu = "#block_top_menu > ul > li:nth-child(2) > a";
+    this.menuContentPageURL = "https://www.saucedemo.com/";
   }
 
   public visitMenuContentPage(): void {
     cy.visit(this.menuContentPageURL);
+    cy.get("input[id='user-name']").type("standard_user"); //(2)
```

Comment:
It would be good to the selectors as variables

Reply:
Menu-Content-page file was deleted